qcli/bsp2svg/converter.py

```diff
@@ -31,52 +31,67 @@ def convert(bsp_file, svg_file, args):
     """
     print(f'Reading {os.path.basename(bsp_file)}')
     bsp_file = Bsp.open(bsp_file)
+    projection_axis = args.projection_axis
+    
+    # Filter faces
+    faces = [face for model in bsp_file.models for face in model.faces]
+    ignore_textures = ['clip', 'hint', 'trigger'] + args.ignore
+    faces = list(filter(lambda f: not (f.texture_name.startswith('sky') or f.texture_name in ignore_textures), faces))
 
     # Determine map bounds
-    vs = [vertex[:] for model in bsp_file.models for face in model.faces for vertex in face.vertexes]
+    vs = [vertex[:] for face in faces for vertex in face.vertexes]
     xs = [v[0] for v in vs]
     ys = [v[1] for v in vs]
+    zs = [v[2] for v in vs]
 
-    min_x = min(xs)
-    max_x = max(xs)
-    min_y = min(ys)
-    max_y = max(ys)
+    drawing_xs = xs if projection_axis in ['y', 'z'] else ys
+    drawing_ys = zs if projection_axis in ['x', 'y'] else ys
+    drawing_min_x = min(drawing_xs)
+    drawing_max_x = max(drawing_xs)
+    drawing_min_y = min(drawing_ys)
+    drawing_max_y = max(drawing_ys)
 
-    width = max_x - min_x
-    height = max_y - min_y
+    width = drawing_max_x - drawing_min_x
+    height = drawing_max_y - drawing_min_y
     padding = min(width // 10, height // 10)
 
     dwg = svgwrite.Drawing(
         svg_file,
-        viewBox=f'{min_x - padding} {min_y - padding} {width + padding * 2} {height + padding * 2}',
+        viewBox=f'{drawing_min_x - padding} {drawing_min_y - padding} {width + padding * 2} {height + padding * 2}',
         profile='tiny'
     )
 
     dwg.add(
         dwg.rect(
-            insert=(min_x - padding, min_y - padding),
+            id='background',
```

Comment:
Interesting. What does this id do?

Reply:
nothing groundbreaking 😅 but it does look prettier when opened with Inkscape, in the "Layers and Objects" section
![image](https://github.com/joshuaskelly/quake-cli-tools/assets/6061770/fa745f6b-417e-436e-9ff7-007018823ad3)

Reply:
Haha. Good enough for me!